source/Body.cpp

```diff
@@ -223,8 +226,11 @@ void Body::LoadSprite(const DataNode &node)
 			frameRate = 1. / child.Value(1);
 		else if(child.Token(0) == "delay" && child.Size() >= 2 && child.Value(1) > 0.)
 			delay = child.Value(1);
-		else if(child.Token(0) == "scale" && child.Size() >= 2 && child.Value(1) > 0.)
-			scale = static_cast<float>(child.Value(1));
+		else if(child.Token(0) == "scale" && child.Size() == 2 && child.Value(1) > 0.)
```

Comment:
~~smh this is what happens when you rewrite my perfect suggestions intead of accepting them~~
```suggestion
		else if(child.Token(0) == "scale" && child.Size() >= 2 && child.Value(1) > 0.)
```